index.js

```diff
@@ -1 +1,4 @@
 // code away!
+const server = require('./server');
+
+server.listen(6000, () => console.log('Up And Running'));
```

Comment:
I recommend indicating the port number in your console log here! This is easier to do if you assign it to a variable `port` and pass that in instead of the number alone.